public/components/TagContext/TagRelationship/TagRelationship.react.js

```diff
@@ -100,6 +101,10 @@ export default class TagRelationship extends React.Component {
         return false;
       }
 
+      if (this.props.tag.type !== tagTypes.topic.name) {
```

Comment:
I've seen topics being mapped to keyword tags. Does this mean this rule should also compare to keyword tags?